### PR TITLE
Replacing soup with lxml

### PR DESCRIPTION
**Pull request by CodyCBakerPhD**
In response to `nwb-converion-tools` [issue #58](https://github.com/catalystneuro/nwb-conversion-tools/issues/58), as well as the reversion of module reliance back to `to_nwb`, I'll be going through these helper functions and replacing usage of soup with direct lxml access. I'll post again later when all the changes are complete.

### CODE REVIEW

File: to_nwb/neuroscope.py [hunk 1]
```diff
@@ -6,25 +6,26 @@
...
 from pynwb.ecephys import ElectricalSeries, LFP, SpikeEventSeries, EventWaveform#, UnitSeries
 from hdmf.backends.hdf5.h5_utils import H5DataIO
 from hdmf.data_utils import DataChunkIterator
 from pynwb.misc import AnnotationSeries
 from tqdm import tqdm
+from typing import Union
 
 from .utils import check_module
 
+PathType = Union[str, Path, None]
```
Comment (bendichter) [inline to_nwb/neuroscope.py]: A better way to allow for `None` is to define this as 

```python
Path = Union[str, Path]
```

and then when you want to allow `None`, use 

```python
from typing import Optional
def my_func(opt_str: Optional[Path])
    pass
```

Comment (CodyCBakerPhD) [reply to_nwb/neuroscope.py]: Good to know! The `Path = Union[str, Path, None]` is what we currently use in `SpikeInterface` so if we change it here we can also change it over there.

That said, I think I might save all the `Path` usage for a different branch/PR and just leave this one for the `lxml` changes, but I'll definitely use that trick whenever I can.

File: to_nwb/neuroscope.py [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
 import os
 from glob import glob
 
 import numpy as np
 import pandas as pd
-from bs4 import BeautifulSoup
+from lxml import etree as et
 from pynwb.behavior import SpatialSeries
-from pynwb.ecephys import ElectricalSeries, LFP, SpikeEventSeries, EventWaveform#, UnitSeries
+from pynwb.ecephys import ElectricalSeries, LFP, SpikeEventSeries
```
Comment (CodyCBakerPhD) [inline to_nwb/neuroscope.py:11]: These were unused

**Comment by CodyCBakerPhD** [comment]
@bendichter OK, this PR should be ready. All soup usage has been replaced. I also removed some unused imports, and enforced PEP8 (which my system can now detect) on `neuroscope.py` and `setup.py`

Oh, also, what standard do we like to use for docstrings? SpikeInterface looked like it may have been using Numpy, whereas these seem to use PEP 257?

**Comment by bendichter** [comment]
@CodyCBakerPhD I prefer numpy formatted strings

**Pull request merged**